src/base/cubrid_getopt.h

```diff
@@ -77,5 +83,8 @@ extern "C"
 #ifdef __cplusplus
 }
 #endif
+#endif
+
+#endif				/* !_CUBRID_GETOPT_H_ */
 
-#endif				/* !_GETOPT_H_ */
+typedef struct option GETOPT_LONG;
```

Comment:
I think it should be inside of the _CUBRID_GETOPT_H_ block.

Reply:
yes, you are right